regexes/device/mobiles.yml

```diff
@@ -5066,10 +5099,16 @@ Samsung:
       model: 'GALAXY J7+ (C7)'
     - regex: '(?:SAMSUNG-)?SM-G610[FMY]'
       model: 'GALAXY J7 Prime'
+    - regex: '(?:SAMSUNG-)?SM-G615[F]'
+      model: 'GALAXY J7 Max'
     - regex: '(?:SAMSUNG-)?SM-J730(?:F|G|GM)'
       model: 'GALAXY J7 Pro'
+    - regex: '(?:SAMSUNG-)?SM-J701(?:F|M)'
+      model: 'GALAXY J7 Nxt/Neo/Core'
```

Comment:
would use one of the names only here as well

Reply:
Same as the other comment: I don't disagree, but don't know which one could be best...
The names for this one seem to be different by region:
* Nxt in India
* Neo in LATAM
* Core in Philippines